shared_model/interfaces/transaction_responses/stateful_failed_tx_response.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_STATEFUL_FAILED_TX_RESPONSE_HPP
+#define IROHA_STATEFUL_FAILED_TX_RESPONSE_HPP
+
+#include "interfaces/primitive.hpp"
+#include "model/transaction_response.hpp"
+#include "utils/string_builder.hpp"
+
+namespace shared_model {
+  namespace interface {
+
+    /**
+     * Tx response of broken stateful validation
+     */
+    class StatefulFailedTxResponse
```

Comment:
Maybe use macros? This code is supa-pupa-overduplicated D:
e.g:
`#define TX_RESPONCE_CLASS(class_name, old_status) {..class code..}`
and use:
```cpp
TX_RESPONCE_CLASS(StatefulFailedTxResponse, iroha::model::TransactionResponse::Status::STATEFUL_VALIDATION_FAILED);
TX_RESPONCE_CLASS(...);
etc
```
It definitely will be lesser copy-paste errors

Reply:
*note: template hackish solution looks ugly, macros are good to use here imo

Reply:
I guess macro in the current solution not so good because it will be broken when some classes need to be extended. Also, this approach decreases readability at all.